brand/migrations/0041_embracecampaign.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.1.7 on 2024-03-26 18:22
+
+from django.db import migrations, models
```

Comment:
When putting together a commit with migrations, it's best to squash your commits. The easiest way to achieve this is to delete the migration files locally and `python manage.py make migrations` another time. Then they'll all be in one file